### PR TITLE
schemahcl: replace schemahcl.Value with cty.Value

### PR DESCRIPTION
**Pull request by a8m**

### CODE REVIEW

File: schemahcl/hcl_test.go [hunk 1]
```diff
@@ -361,83 +361,6 @@ func TestMultiFile(t *testing.T) {
 	}, test.People[1])
 }
 
-func TestDynamicBlocks(t *testing.T) {
```
Comment (rotemtam) [inline schemahcl/hcl_test.go:364]: is this added elsewhere?

File: schemahcl/opts.go [hunk 2]
```diff
@@ -188,28 +188,14 @@ func typeFuncSpecImpl(_ *function.Spec, typeSpec *TypeSpec) function.ImplFunc {
...
-				lst := &ListValue{}
-				for _, arg := range args {
-					v, err := extractValue(arg)
-					if err != nil {
-						return cty.NilVal, err
-					}
-					lst.V = append(lst.V, v)
-				}
-				t.Attrs = append(t.Attrs, &Attr{K: attr.Name, V: lst})
+				t.Attrs = append(t.Attrs, &Attr{K: attr.Name, V: cty.ListVal(args)})
```
Comment (rotemtam) [inline schemahcl/opts.go:191]: nice

File: schemahcl/hcl_test.go (hunk 1 above)
Comment (a8m) [reply schemahcl/hcl_test.go:364]: No. I decided to remove it as it's not needed anymore, and it just adds complexity to the current package.

**Review by rotemtam — APPROVED** [review]

**Pull request merged**